docs/INSTALLING.md

```diff
@@ -0,0 +1,71 @@
+# How do I install the Code Corps Ember app?
+
+### Requirements
+
+You will need to install the following: 
+
+- [Node.js/npm](https://docs.npmjs.com/getting-started/installing-node) we use the LTS version (v 4.4.7)
+- bower: `npm install -g bower`
+- ember-cli: `npm install -g ember-cli`
+
+Do you use other versions of node.js? Check out [nvm](https://github.com/creationix/nvm)
+
+## Clone this repository
+
+You'll want to [clone this repository](https://help.github.com/articles/cloning-a-repository/) with `git clone https://github.com/code-corps/code-corps-ember.git`. If you plan on contributing, you'll want to fork it too!
+
+
+The directory structure will look like the following:
+
+```shell
+code-corps-ember/    # → Root folder for this project
+├── app/
+├── config/
+├── dist/
+├── ...              # → More standard Ember.js files
+...
+```
```

Comment:
We could definitely make use of this. Things like `abilities` which are non-standard, for example, could be nice to point out in here. The more introduction to how our directories are laid out and where to find important things, the better, IMO.